Design.md

```diff
@@ -0,0 +1,52 @@
+# Design
+ 
+## Senior UI/UX Designer
+- I have at least 5 years experience
```

Comment:
See https://www.quora.com/Is-it-correct-to-say-Five-years-experience-or-Five-years-experience

1. Fix this one and the others